tests/Doctrine/Tests/ORM/Functional/Ticket/GH5762Test.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @group GH-5762
+ */
+class GH5762Test extends \Doctrine\Tests\OrmFunctionalTestCase
+{
+
+    protected function setup()
+    {
+        parent::setup();
+
+        $this->_schemaTool->createSchema(array(
+            $this->_em->getClassMetadata(__NAMESPACE__ . '\GH5762Driver'),
```

Comment:
You could use `::class` constant for all of these classes now, since the minor PHP version currently allowed is 5.6.